analyses/cell-type-wilms-tumor-06/render_01-clustering.R

```diff
@@ -0,0 +1,92 @@
+# This script is to run the 01-clustering.Rmd template notebook over all samples
+
+## Load libraries
+
+library(SingleCellExperiment)
+library(Seurat)      # to use Seurat workflow
+
+library(tidyr)
+library(dplyr)
+library(DT)          # for table visualization
+
+library(ggplot2)     # for visualization
+library(patchwork)   # for visualization
+library(ggplotify)
+library(SCpubr)      # for visualization
+library(assertthat)  # for visualization, to be able to use do_DimPlot
+library(viridis)     # for visualization - colors
+
+library(edgeR)       # For pseudobulk based differential expression (DE) analysis
+library(DElegate)    # for pseudobulk based DE analysis and identification of marker genes
+
+library(Azimuth)     # For label transfer
+
+library(msigdbr)     # for gene set enrichment analysis or pathway enrichment
+library(clusterProfiler)
+
+library(data.table)
+
+## define some config set-up / threshold
+
+# We store in a config list names "cfg" parameters used all along the analysis to filter for p-value, log fold change, percentage of expression, etc.
+
+cfg = list()
+cfg$padj_thershold = 0.05
+cfg$lfc_threshold = 1
+cfg$rate1_threshold = 0.5
+set.seed(12345)
+
+
+## Define the base directories
+
+# The base path for the OpenScPCA repository, found by its (hidden) .git directory
+repository_base <- rprojroot::find_root(rprojroot::is_git_root)
+
+# The current data directory, found within the repository base directory
+data_dir <- file.path(repository_base, "data", "2024-07-08", "SCPCP000006")
```

Comment:
We'd like to use `current` here FYI, not the date; see the "tip" here: https://openscpca.readthedocs.io/en/latest/getting-started/accessing-resources/getting-access-to-data/#downloaded-data-file-structure

```suggestion
data_dir <- file.path(repository_base, "data", "current", "SCPCP000006")
```